examples/live/run.sh

```diff
@@ -195,13 +195,26 @@ python -m mpi4py `which pycbc_live` \
 --src-class-eff-to-lum-distance 0.74899 \
 --src-class-lum-distance-to-delta -0.51557 -0.32195 \
 --run-snr-optimization \
+--snr-opt-di-maxiter 50 \
+--snr-opt-di-popsize 100 \
+--snr-opt-include-candidate \
+--snr-opt-seed 42 \
```

Comment:
I would remove this option, as it is equal to the default and IMO we prefer to use default values for checking

Reply:
(i.e. if at some point something bad happens to the default, we want the test to be sensitive to it)

Reply:
I can remove it, should I also remove the commented out options which indicate how to use particle swarm optimization?

Reply:
I included them to make the example a bit more helpful for those that would like to test and use the pso optimizer (as it is the superior optimizer).